internal/pkg/plugin/githubactions/golang/update.go

```diff
@@ -8,11 +8,11 @@ import (
 	"github.com/merico-dev/stream/pkg/util/github"
 )
 
-// Reinstall remove and set up GitHub Actions workflows.
-func Reinstall(options *map[string]interface{}) (bool, error) {
+// Update remove and set up GitHub Actions workflows.
+func Update(options *map[string]interface{}) (map[string]interface{}, error) {
```

Comment:
Is it possible to call "Delete" first then "Create"? If so, the "Update" implementation would have only 2 single lines of code which is easier to read and has less duplicated code.

Reply:
Same for other plugins. Maybe it's not possible; just to think about it.

Reply:
because "Delete"  and "Create" are both include  "parseAndValidateOptions(options)" and "NewClient" operations, they are repetitive. so I merge the public operations into "Update", instead of using "Delete" + "Create"